README-dnsrecord.md

```diff
@@ -0,0 +1,258 @@
+DNSRecord module
+================
+
+Description
+-----------
+
+The dnsrecord module allows management of DNS records and is as compatible as possible with the Ansible upstream `ipa_dnsrecord` module, but provide some other features like multiple record management in one execution and support for more DNS record types.
+
+
+Features
+--------
+* DNS record management.
+
+
+Supported FreeIPA Versions
+--------------------------
+
+FreeIPA versions 4.4.0 and up are supported by the ipadnsrecord module.
+
+
+Requirements
+------------
+
+**Controller**
+* Ansible version: 2.8+
+
+**Node**
+* Supported FreeIPA version (see above)
+
+
+Usage
+=====
+
+Example inventory file
+
+```ini
+[ipaserver]
+ipaserver.test.local
+```
```

Comment:
I think it would be a good idea to tell the users know they actually have to create the DNS Zones prior to creating the records. While it might seem obvious it's pretty common that ansible things deal with stuff like that. Since we are not dealing I would just add the link to dnszone module and/or a very basic example on how to use it.

Reply:
Could be, but we have other modules with similar behavior that do not have this.

Maybe it is time to start adding something like that. ;-)